frontend/shared/src/i18n/i18n.ts

```diff
@@ -0,0 +1,10 @@
+export type Language = 'fi' | 'sv' | 'en';
+export type I18nNamespace = 'common';
+
+export const DEFAULT_LANGUAGE: Language = 'fi';
```

Comment:
if you refer to SUPPORTED_LANGUAGES on line 5, it is array because they are always used as an array of values. If you check the same file for `benefit` side it has the enum but everywhere where is it used, it is wrapped up with Object.values(SUPPORTED_LANGUAGES) to get the array of values from the enum.

Reply:
I mean `Language` could be: 
```Typescript
export enum Language {
  FI = 'fi',
  SV = 'sv',
  EN = 'en,
}

export const DEFAULT_LANGUAGE = Language.FI;
```
This is just a nitpick :)